client/client.go

```diff
@@ -63,6 +63,25 @@ const (
 	TaskReachabilityClosedWorkflows = internal.TaskReachabilityClosedWorkflows
 )
 
+// UpdateLifeCycleStage indicates the stage of an update request.
+// NOTE: Experimental
+type UpdateLifeCycleStage = internal.UpdateLifeCycleStage
```

Comment:
Can we call this `WorkflowUpdateLifeCycleStage` just to disambiguate any future uses of the general term "update"?